people/people.yml

```diff
@@ -41,6 +41,23 @@ Taniya Das:
     <name> is a Research Engineer working on various engineering and machine learning tasks for <a href="https://github.com/openml" target="_blank">openml.org</a> and  <a href="https://github.com/aiondemand/AIOD-rest-api" target="_blank">AI-on-Demand</a> platform (a EU project), and is currently thinking of ML methods to make OpenML better. 
   picture: ../images/people/taniya_image.webp
 
+Joaquin Vanschoren:
+  position: permanent
+  description:
+    Joaquin Vanschoren is Associate Professor at the Eindhoven University of Technology. His research focuses on deeply understanding 
+    machine learning algorithms and how AI can make the world a better place. He believes in democratizing AI by turning insights into
+    progressively more automated and efficient methods by combining AutoML, meta-learning, and continual learning, and by thoroughly 
+    benchmarking their real-world performance and safety. <br />
+    He founded and leads OpenML.org, an open platform for sharing machine learning resources and experiments, initiated the NeurIPS 
+    Datasets and Benchmarks track to create new incentives to focus research on data and benchmarks, is editor-in-chief of the Journal 
+    for Data-Centric Machine Learning (DMLR, part of JMLR), and co-chairs the MLCommons Working Group on AI Safety evaluation. <br />
+    He won the Dutch Data Prize, an Amazon Research Award, and an ECMLPKDD Best Demo award. He gave over 30 invited talks, was a 
+    tutorial speaker at NeurIPS 2018 and AAAI 2021, and authored over 150 scientific papers, as well as reference books on 
+    Automated Machine Learning and Meta-learning. He is action editor of JMLR and moderator for ArXiv. He co-organized the NeurIPS 
+    and ICML workshops on meta-learning and AutoML, as well as the AutoML and SysML conferences. He is a founding member of the 
+    European AI networks ELLIS and CLAIRE.
+  picture: ../images/people/joaquin.webp
```

Comment:
You also need the image. It's not included in this PR. If you want you can go ahead and merge this PR so your text is already there, and add the picture another time.

Reply:
@joaquinvanschoren ping